myfleetmanager/catalog/views.py

```diff
@@ -245,6 +245,76 @@ def user_detail(request, user_id):
     }
     return render(request, 'user_management/user_detail.html', context)
 
+@login_required
+@user_passes_test(is_admin)  # Ensure only admins can access this view
+def delete_feedback(request, feedback_id):
+    feedback = get_object_or_404(Feedback, pk=feedback_id)
+    if request.method == "POST":
+        feedback.delete()
+        return redirect('feedback_list')  # Redirect to the feedback list or another appropriate page
+    return render(request, 'page_management/delete_feedback.html', {'feedback': feedback})
+
+
+@login_required
+@user_passes_test(lambda user: user.groups.filter(name='Admin').exists())
+def resolve_feedback(request, feedback_id):
+    feedback = get_object_or_404(Feedback, pk=feedback_id)
+    feedback.resolved = True
+    feedback.save()
+    return redirect('feedback_list')  # Redirect back to the feedback list
+
+
+@login_required
+@user_passes_test(lambda user: user.groups.filter(name='Admin').exists())
+def feedback_list_view(request):
+    users = User.objects.all()
+    categories = Feedback.CATEGORY_CHOICES
+    selected_user = request.GET.get('user', None)
+    selected_category = request.GET.get('category', None)
+
+    feedback_queryset = Feedback.objects.all()
+
+    if selected_user:
+        try:
+            selected_user = int(selected_user)
+            feedback_queryset = feedback_queryset.filter(user_id=selected_user)
+        except ValueError:
+            selected_user = None
+
+    if selected_category:
+        feedback_queryset = feedback_queryset.filter(category=selected_category)
+
+    # Separate feedback into resolved and unresolved
+    unresolved_feedback = feedback_queryset.filter(resolved=False)
+    resolved_feedback = feedback_queryset.filter(resolved=True)
+
+    context = {
+        'unresolved_feedback': unresolved_feedback,
+        'resolved_feedback': resolved_feedback,
+        'users': users,
+        'categories': categories,
+        'selected_user': selected_user,
+        'selected_category': selected_category,
+    }
+
+    return render(request, 'page_management/feedback_list.html', context)
+
+
+@login_required
+@user_passes_test(is_admin)  # Ensure only admins can access this view
+def edit_footer_content(request):
+    footer_content, created = FooterContent.objects.get_or_create(pk=1)  # Assuming only one instance is needed
+
+    if request.method == "POST":
+        form = FooterContentForm(request.POST, instance=footer_content)
+        if form.is_valid():
+            form.save()
+            return redirect('admin_dashboard')  # Redirect after saving
+    else:
+        form = FooterContentForm(instance=footer_content)
+
+    return render(request, 'page_management/edit_footer_content.html', {'form': form})
```

Comment:
These weren't in a spot that made sense organizationally so they got moved up here